### PR TITLE
Site refactor

### PR DESCRIPTION
**Pull request by jmertic**

### CODE REVIEW

File: process/contributing.md [hunk 1]
```diff
@@ -103,6 +103,40 @@ A: There is no requirement for a project to adopt [ASF Contributor Agreements] w
...
+the red EasyCLA "NOT COVERED" error. What do I do to get the error
+to go away?
+
+A: Refer to the [EasyCLA documentation](https://docs.linuxfoundation.org/lfx/easycla/v2-current/getting-started) on making your first contribution to a EasyCLA managed repository, but generally it should go away on it's own after a few minutes. If you still have issues, please file a [support request].
+
+Q: My company has a corporate CLA for another ASWF project, doesn't that cover this project, too?
+
+A: Each ASWF project leverages a seperate CLA for contributions to thier project. You can follow the [guide](https://docs.linuxfoundation.org/lfx/easycla/v2-current/contributors/corporate-contributor) to automatically notify your organization's CLA manager for the given project and authorize you under the CCLA.
+
+Q: My corporate CLA manager says I should be on the approved list, but my PR is still blocked. What gives?
```
Comment (jfpanisset) [inline process/contributing.md:136]: Corporate CLA capitalization

**Review by jfpanisset — DISMISSED** [review]
All good, feel free to merge as is and fix up issues in follow up PR, or add another commit.

File: .github/workflows/updateprojects.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3                                                                                                  
+#                                                                                                                       
+# Copyright this project and it's contributors
```
[the remark this answers is not in the document]
Comment (jmertic) [reply .github/workflows/updateprojects.py]: I love that you even do editoral checks on the Copyright statements ;-)

File: engagement/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+## TAC Meetings
+
+Learn more about joining the public meetings of the Technical Advisory Council (TAC) on [the Meetings page](https://tac.aswf.io/meetings)
+
+## Mailing Lists and Slack channels for hosted projects
+
+Each project hosted at the Academy Software Foundation collaborates on open channels that are welcome for anyone in the community to participate in. See the below list of channels for each project.
+
+All Slack channels referenced below are part of the [Academy Software Foundation Slack organization](https://slack.aswf.io)
```
[the remark this answers is not in the document]
Comment (jmertic) [reply engagement/README.md:19]: Good point - I will fix that one manually.

File: .github/workflows/updateprojects.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+projectsCsvFile = os.path.dirname(os.path.realpath(__file__))+'/../../_data/projects.csv'
+
+landscapeBaseURL = 'https://landscape.aswf.io'
+landscapeHostedProjects = landscapeBaseURL+'/data/exports/projects-hosted.json'
+landscapeSingleItem = landscapeBaseURL+'/data/items/{}.json'
+
+# map landscape ID to slug
+keyMapping = {
+    'open-mainframe-education': 'mainframe-open-education'
```
Comment (jfpanisset) [inline .github/workflows/updateprojects.py:21]: Does this need to be updated for ASWF?

**Review by jfpanisset — DISMISSED** [review]
Looks good, updateprojects.py may need ASWF-specific update

File: .github/workflows/updateprojects.py (hunk 4 above)
Comment (jmertic) [reply .github/workflows/updateprojects.py]: Yep - actually removed entirely as it's not used. Done now

**Review by jfpanisset — APPROVED** [review]
Looks good

**Pull request merged**